lib/gstvkvideoparser.cpp

```diff
@@ -117,14 +117,14 @@ gst_video_parser_constructed (GObject * object)
   if (self->codec == VK_VIDEO_CODEC_OPERATION_DECODE_H264_BIT_EXT) {
     parser_name = "h264parse";
     src_caps_desc = "video/x-h264,stream-format=byte-stream";
-    decoder = g_object_new (GST_TYPE_H264_DEC, "user-data", self->user_data,
-        "oob-pic-params", self->oob_pic_params, NULL);
+    decoder = static_cast<GstElement*>(g_object_new (GST_TYPE_VK_H264_DEC, "user-data", self->user_data,
```

Comment:
use `reinterpret_cast` for these object type conversion

Reply:
also, these type conversion might be moved to a windows compilation related commit